runtime/src/runtime/fvm.rs

```diff
@@ -12,7 +12,6 @@ use fvm_shared::crypto::signature::Signature;
 use fvm_shared::econ::TokenAmount;
 use fvm_shared::error::{ErrorNumber, ExitCode};
 use fvm_shared::piece::PieceInfo;
-use fvm_shared::randomness::Randomness;
 use fvm_shared::sector::{
```

Comment:
All the types that implement sector or storage-specific functions can also be removed.